CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+## Welcome to the TJ-Bot project!
+
+First off, thank you for considering contributing to TJ-Bot. :tada:
+
+TJ-Bot is an open source project, and we love to receive contributions from our community — you! There are many ways to contribute, from writing tutorials or blog posts, improving the documentation, submitting bug reports and feature requests or writing code which can be incorporated into TJ-Bot itself.
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+
+## Ground Rules
+
+* Create issues for any major changes and enhancements that you wish to make. Discuss things transparently and get community feedback.
+* Be welcoming to newcomers and encourage diverse new contributors from all backgrounds.
+
+
+## Your First Contribution
+
+Unsure where to begin contributing to TJ-Bot? You can start by looking through these labels!
+* [good first issue](https://github.com/Together-Java/TJ-Bot/issues/?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) - issues which should only require a few lines of code, and a test or two.
+* [help wanted](https://github.com/Together-Java/TJ-Bot/issues/?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)  - issues which should be a bit more involved than good first issues.
+
+Working on your first Pull Request? You can check these resources:
+* http://makeapullrequest.com/
+* http://www.firsttimersonly.com/
+
+
+At this point, you're ready to make your changes! Feel free to ask for help; everyone is a beginner at first! :tada: 
+
+
+## Getting started
+
+1. Create your own fork of the repository.
+2. Run the tests. We only take pull requests with passing tests, and it's great to know that you have a clean slate.
+3. To comply with our style guidelines use `./gradlew spotlessApply`. PRs should not include commits with autostyle runs. Instead, fix any style violations before committing changes.
```

Comment:
Obviously, this is assuming the tech stack we will use, so I guess we will just edit it later if this prediction is wrong.

Reply:
Yeah. Everything is WIP now. Lets take it and edit things later.